ffv1.md

```diff
@@ -429,24 +429,60 @@ Early experimental versions of FFV1 used the CABAC Arithmetic coder from H.264 a
 
 #### Range Binary Values
 
-To encode binary digits efficiently a Range coder is used.
+To encode binary digits efficiently a Range coder is used. A Range coder encodes a series of symbols by using a probability estimation for each symbol (expressed in Quantization Table Sets for an FFV1 encoding) and a range of values that can be divided into sub-ranges proportional to the probability of each symbol encoded. The range is sub-divided until a sub-range can be used to decode the symbol.
+
+To describe Range coding for FFV1 the following values are used:
 
-{spacing="compact"}
 C~i~
 : the i-th Context.
 
 B~i~
 : the i-th byte of the bytestream.
 
+R~i~
+: the Range at the i-th symbol.
+
+r~i~
+: the boundary between two sub-ranges of R~i~: a sub-range of r~i~ values and a sub-range R~i~ - r~i~ values.
+
+L~i~
+: the Low value of the Range at the i-th symbol.
+
+t~i~
+: a temporary variable to transmit sub-ranges between range coding operations.
+
 b~i~
 : the i-th Range coded binary value.
 
 S~0,\ i~
 : the i-th initial state.
 
-j_(n)
+j~n~
 : the length of the bytestream encoding n binary symbols.
 
+The following Range coder state variables are initialized to the following values. The Range is initialized to a value of 65,280 (expressed in base 16 as 0xFF00) as depicted in [@figureInitializeRange]. The Low is initiatlized according to the value of the first two bytes as depicted in [@figureInitializeLow]. The value of j~n~ which tracks the length of the bytestream is initialized to 2 as depicted in [@figureInitializeLength].
+
```

Comment:
typo: `initiatlized`

Reply:
It is outside of this PR, but I added a new commit for it. https://github.com/FFmpeg/FFV1/pull/259/commits/2bb2b48fc81186ef29712983cc113daff4af0279

Reply:
ah, i'm wrong this sentence is introduced earlier in the pr, thx